### PR TITLE
Feature/add get culture practice

### PR DESCRIPTION
**Pull request by NikitaChyzhykDev4agro**

### CODE REVIEW

File: Dev4Agriculture.ISO11783.ISOXML/Analysis/ISODeviceAnalysis.cs [hunk 1]
```diff
@@ -1,5 +1,7 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
+using de.dev4Agriculture.ISOXML.DDI;
 using Dev4Agriculture.ISO11783.ISOXML.IdHandling;
```
Comment (Frank-Wiebeler) [inline Dev4Agriculture.ISO11783.ISOXML/Analysis/ISODeviceAnalysis.cs:5]: Remove unrequired usings please ;)

File: Dev4Agriculture.ISO11783.ISOXML/Constants.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@ public class Constants
     {
         //TODO It's more save to move this Element to a common class. It's found in Generator and DotnetCore Library
         public static string ISOXMLClassName = "Dev4Agriculture.ISO11783.ISOXML";
-        public static string Version = "V0.17.1.1";//Currently unused; Just for the commit
+        public static string Version = "V0.17.2";//Currently unused; Just for the commit
```
Comment (Frank-Wiebeler) [inline Dev4Agriculture.ISO11783.ISOXML/Constants.cs:7]: As version 0.17.2 was already released, we need 0.17.2.1.

**Review by Frank-Wiebeler — CHANGES_REQUESTED** [review]
Just minor topics left 👍

File: Dev4Agriculture.ISO11783.ISOXML/Analysis/ISODeviceAnalysis.cs (hunk 1 above)
Comment (NikitaChyzhykDev4agro) [reply Dev4Agriculture.ISO11783.ISOXML/Analysis/ISODeviceAnalysis.cs:5]: image
It's require to use DDIList

**Pull request closed without merging**